client/js/store/map/Layers.js

```diff
@@ -200,15 +200,34 @@ const LayersStore = {
     get ({ commit, dispatch }, procedureId) {
       commit('setProcedureId', procedureId)
 
-      return dpApi({
-        method: 'GET',
-        url: Routing.generate('dplan_api_procedure_layer_list',
-          {
-            procedureId: procedureId,
-            include: ['categories', 'gisLayers'].join()
-          }
-        )
-      })
+      return dpApi.get(Routing.generate('api_resource_list', {
```

Comment:
this call returns a 400. I don't know if it is intended to be like this here and if it is fixed in other PRs. I am a bit lost... The request looks good to me